plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -275,6 +282,7 @@ func (s *SQLServer) gatherServer(server string, query Query, acc telegraf.Accumu
 		conn = sql.OpenDB(connector)
 	}
 
+	// deferred connection closure
```

Comment:
Please remove as it does not add any information.

Reply:
done